src/views/LogsView.js

```diff
@@ -58,26 +59,32 @@ const LogsView = ({events: {loading, allEvents, error, refetch}, user}) => {
     );
   if (isAdmin) {
     return (
-      <Container as={Segment} basic vertical>
-        <Segment basic floated="right" className="noMargin noPadding">
-          <Select
-            clearable
-            placeholder="Event Type"
-            options={eventTypeOptions}
-            onChange={(e, {name, value}) => refetch({eventType: value})}
-          />
-        </Segment>
-        <Header as="h2" className="mt-6">
-          Event Logs
-        </Header>
-        {allEvents.edges.length > 0 ? (
-          <EventList events={allEvents.edges} />
-        ) : (
-          <Header disabled textAlign="center" as="h4">
-            No event logs available or match your filter option.
+      <AnalyticsViewConsumer
+        mountProperties={{
+          events_length: allEvents.edges.map(({node}) => node).length,
```

Comment:
Since this is the "Logs" tab under study, do we want to include study kfid and name here?